test_runner/regress/test_disk_usage_eviction.py

```diff
@@ -230,6 +230,9 @@ def _eviction_env(
     neon_env_builder.num_pageservers = num_pageservers
     neon_env_builder.enable_pageserver_remote_storage(RemoteStorageKind.LOCAL_FS)
 
+    # Disable compression support for EvictionEnv to get larger layer sizes
+    neon_env_builder.pageserver_config_override = "image_compression='disabled'"
+
```

Comment:
This makes sense; we cannot really predict how the compression influences the sizes and I really wouldn't want to. The comparisons are showing how much of the layers remains, so this is the best way forward sanity-wise.

It does raise the question of should we do more with eviction and layer file sizes... Currently all layers are equally weighted.